src/app/profiles/profile-form.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
+import { constants } from './profile-constants';
+
+@Component({
+  selector: 'app-profile-form',
+  templateUrl: './profile-form.html',
+})
+export class ProfileFormComponent {
+  @Input() services: {
```

Comment:
I think @Component() decorator would prefer a class, otherwise an interface definition would be like passing an empty object?  Also if I used the angular-cli to generate the component, it would define a class for the component.